src/navigation/ActivationStack.tsx

```diff
@@ -119,4 +94,23 @@ const ActivationStack: FunctionComponent = () => {
   )
 }
 
+export const applyHeaderLeftBackButton = () => {
```

Comment:
would we be able to use the common component for this one? not sure that we should, but irc we added this to updated the back behavior, which we no longer need to do.

Reply:
Ah good catch!